File: node_npm_examples/websocket/tickerWebsocket.js
const ba = require('bitcoinaverage');

var publicKey = 'yourPublicKey';
var secretKey = 'yourSecretKey';
var ws = ba.websocketClient(publicKey, secretKey);

// Connecting to the local ticker and printing BTCEUR price data; you can try it with 'global'
ws.connectToTickerWebsocket('local', 'BTCEUR', function (response) {
    console.log(JSON.stringify(response, null, 4));
});

File: python_pip_examples/websocket/ticker_websocket.py
from bitcoinaverage import TickerWebsocketClient

if __name__ == '__main__':
    secret_key = '' or input('Enter your secret key: ')
    public_key = '' or input('Enter your public key: ')

    print('Connecting to the ticker websocket...')
    ws = TickerWebsocketClient(public_key, secret_key)
    ws.ticker_data('local', 'BTCUSD')
